scripts/world_bank/datasets/datasets.py

```diff
@@ -131,19 +139,21 @@ class Mode:
 
 def download_datasets():
     '''Downloads dataset files. This is a very expensive operation so run it with care. It assumes that the datasets CSV is already available.'''
-
+    logging.info('start download_datasets')
     with open(DATASETS_CSV_FILE_PATH, 'r') as f:
         csv_rows = list(csv.DictReader(f))
         download_urls = []
         for csv_row in csv_rows:
             download_url = csv_row.get(DATASET_DOWNLOAD_URL_COLUMN_NAME)
             if download_url:
                 download_urls.append(download_url)
-
+    try:
         with multiprocessing.Pool(POOL_SIZE) as pool:
             pool.starmap(download, zip(download_urls))
 
         logging.info('# files downloaded: %s', len(download_urls))
+    except Exception as e:
```

Comment:
Should this exception be ignored at runtime? If it creates incomplete downloads, consider implementing logging.fatal.